ci-operator/config/stackrox/collector/stackrox-collector-master.yaml

```diff
@@ -1,8 +1,17 @@
+base_images:
+  centos:
+    name: centos
+    namespace: origin
+    tag: stream8
+  collector-test:
+    name: collector-test
+    namespace: stackrox
+    tag: latest
 build_root:
   image_stream_tag:
-    name: release
-    namespace: openshift
-    tag: golang-1.17
+    name: collector-test
+    namespace: stackrox
+    tag: latest
```

Comment:
TAL at https://github.com/stackrox/stackrox/pull/1874 for an alternate way to specify the build_root. It allows external images to be used and fits better with our rox-ci-image dependency. Though that can be left for a future commit as it'll need https://github.com/stackrox/rox-ci-image/pull/135 merged and a new rox-ci-image release tagged.